tests/integration/conftest.py

```diff
@@ -144,3 +147,20 @@ async def _factory(evm_address: int, amount: int = PRE_FUND_AMOUNT):
         )
 
     return _factory
+
+
+@pytest.fixture(scope="session")
+def get_starknet_address(account_proxy_class, kakarot):
```

Comment:
this was moved to allow ef_tests and integrations to share this beautiful fixture